frontend/src/components/submission/SubmissionForm.vue

```diff
@@ -53,10 +55,33 @@ const formSchema = object({
     .integer()
     .typeError('Queue Priority must be a number')
     .label('Queue Priority'),
-  updatedAai: string().oneOf(['Y', 'N']).required().label('Updated AAI Spreadsheet')
+  updatedAai: string().oneOf(['Y', 'N']).required().label('Updated AAI Spreadsheet'),
+  user: mixed()
+    .when('intakeStatus', {
+      is: (val: string) => val !== 'SUBMITTED',
+      then: (schema) => schema.required(),
+      otherwise: (schema) => schema.notRequired()
+    })
+    .label('Assignee')
 });
 
 // Actions
+const getAssigneeOptionLabel = (e: any) => {
```

Comment:
Any type here isn't sufficient - should at bare minimum have a typing (even if in-line) that specifies that it expects this to be an object containing at least the `fullName` and `email` attributes. Ideally if we have a type for this construct already we leverage that instead.